Bottom: 2d92ae89cbff3b1eccae9de8b9718649004e2453
Top:    8075f6a47dbad3c54feda08b60166abfced62daf
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-14 16:36:14 -0700

Refresh of dbg-out

---

diff --git a/src/preload.c b/src/preload.c
index efdb414..b2cbb78 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -1076,6 +1076,7 @@ int __fxstat64(int ver, int socket, struct stat64 *buf)
 {
 //	int fd, ret;
 
+	init_preload();
 	fprintf(fout, "%s\n", __func__); fflush(fout);
 //	if (fd_get(socket, &fd) == fd_rsocket) {
 //		ret = real.fxstat64(ver, socket, buf);
@@ -1090,6 +1091,7 @@ int __fxstat64(int ver, int socket, struct stat64 *buf)
 
 int __fxstat(int ver, int socket, struct stat *buf)
 {
+	init_preload();
 	fprintf(fout, "%s\n", __func__); fflush(fout);
 	return real.fxstat(ver, socket, buf);
 }
